saleor/order/utils.py

```diff
@@ -24,6 +24,7 @@
     increase_stock,
 )
 from ..product.utils.digital_products import get_default_digital_content_settings
+from ..shipping.utils import applicable_shipping_methods
```

Comment:
I know that we are doing that in almost all part of code but... Personally - I don't think that we should mixin and import utils file from different modules. It generates circular imports (now or in the future), utils file should contain only helper logic used only internally in the base module.
If you need to import utils's function in a different module it means that the function shouldn't be in the `utils` file

Reply:
Here the idea was to abstract the logic between shipping and checkout/ordering, as shipping is the *base*. So it was safe to assume that there shouldn't have any circular imports happening as shipping shouldn't depend on checkout or order.

But you make a fair point. Maybe we should move it to a class method?